index.js

```diff
@@ -0,0 +1,117 @@
+// Book Class: Represents a Book
+// eslint-disable-next-line max-classes-per-file, import/named
+// import { Book } from './modules/constructor.js';
+// eslint-disable-next-line max-classes-per-file
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+// eslint-disable-next-line max-classes-per-file
+class Store {
+  static getBooks() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+
+    return books;
+  }
+
+  static addBook(book) {
+    const books = Store.getBooks();
+
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
```

Comment:
[OPTIONAL] use array methods that doesn't mutate the array, e.g. [concat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat) instead of push method since it will help to maintain the content of the array rather than mutate or change them. :pray:

Reply:
Thanks, @jerryowusu for the suggestion. We will try it out.